### PR TITLE
Incorporate first round of feedback comments

### PR DESCRIPTION
**Pull request by swiftdiaries**
1. Address first round of feedback comments on installation 
2. Change messaging around k8s installation script

### CODE REVIEW

File: install/README.md [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
 	* Default Kubernetes storage class that can dynamically provision volumes
 
-### [Optional] Prerequisites setup guide
-Here's a [guide](k8sup.md) for creating a test Kubernetes cluster with a dynamic storage class. <br>
-**Note:** This guide is meant for development and testing environments.
+### [Internal] Kubernetes reference
+Here's a  reference [guide](k8sup.md) for creating a test Kubernetes cluster with a dynamic storage class. <br>
+**Note:** This guide is meant for internal development and testing. And serves as a reference for kubeflow-starter-pack users and not as a recommended setup instructions for Kubernetes.
 
 ## Installation
```
Comment (elviraux) [inline install/README.md]: do we even need this section

File: install/README.md [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
+	* Ubuntu 16+
 	* Kubernetes v1.14.x
 	* Default Kubernetes storage class that can dynamically provision volumes
 
-### [Optional] Prerequisites setup guide
-Here's a [guide](k8sup.md) for creating a test Kubernetes cluster with a dynamic storage class. <br>
-**Note:** This guide is meant for development and testing environments.
+### [Internal] Kubernetes reference
+Here's a  reference [guide](k8sup.md) for creating a test Kubernetes cluster with a dynamic storage class. <br>
+**Note:** This guide is meant for internal development and testing. And serves as a reference for kubeflow-starter-pack users and not as a recommended setup instructions for Kubernetes.
```
Comment (elviraux) [inline install/README.md]: and not a part of Cisco Kubeflow starter pack

File: install/README.md [hunk 3]
```diff
@@ -1,24 +1,24 @@
 # Installation Instructions
 
-This directory contains the installation instructions for installing Kubernetes and Kubeflow. 
+This directory contains the instructions for installing Kubeflow. 
 
 ## Prerequisites
 
 - UCS C240
-	* Ubuntu 18.04 LTS
+	* Ubuntu 16+
```
Comment (elviraux) [inline install/README.md]: should we explicitly mention on bare metal?

File: install/README.md [hunk 4]
```diff
@@ -1,24 +1,24 @@
...
 
-This directory contains the installation instructions for installing Kubernetes and Kubeflow. 
+This directory contains the instructions for installing Kubeflow. 
 
 ## Prerequisites
 
 - UCS C240
-	* Ubuntu 18.04 LTS
+	* Ubuntu 16+
 	* Kubernetes v1.14.x
```
Comment (elviraux) [inline install/README.md]: Let's mention that the version is currently required by Kubeflow

File: install/README.md (hunk 3 above)
Comment (swiftdiaries) [reply install/README.md]: If people want to do this on HX VM, it should be possible today.

File: install/README.md (hunk 4 above)
Comment (swiftdiaries) [reply install/README.md]: Added Kubeflow repo and binary versions

File: install/README.md (hunk 3 above)
Comment (elviraux) [reply install/README.md]: is there any concerns why HX VM might be different from bare metal?

**Pull request merged**